### ISSUE TITLE
Insufficient color contrast for text links

### ISSUE THREAD

**Issue by terichadbourne** [ipfs/docs#469]
In working on a fork of this site I discovered that the `$accentColor` that creates the text links on the beta site (both in-line and on the sidebar) doesn't pass contrast checks for accessibility for normal size font on any of the three backgrounds I see it on. Just dropping these reports here (from https://webaim.org/resources/contrastchecker/) in case they're helpful for future accessibility reviews: 

![image](https://user-images.githubusercontent.com/19171465/76878781-b236e780-684b-11ea-9ca9-e9917ef74966.png)

![image](https://user-images.githubusercontent.com/19171465/76878829-c1b63080-684b-11ea-8c2e-ecd33b6425a8.png)

![image](https://user-images.githubusercontent.com/19171465/76878991-fe822780-684b-11ea-8734-a0a652ed9e12.png)

**Comment by johnnymatthews** [comment]
We won't be dealing with this right now, but if anyone wants to have a go a fixing this issue that'd be great!
